github/actions_workflows.go

```diff
@@ -149,13 +149,26 @@ func (s *ActionsService) getWorkflowUsage(ctx context.Context, url string) (*Wor
 	return workflowUsage, resp, nil
 }
 
-// CreateWorkflowDispatchEvent manually triggers a GitHub Actions workflow run.
+// CreateWorkflowDispatchEventByID manually triggers a GitHub Actions workflow run.
 //
-// GitHub API docs: https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event
-func (s *ActionsService) CreateWorkflowDispatchEvent(ctx context.Context, owner, repo string, workflowID int64, event CreateWorkflowDispatchEventRequest) (*Response, error) {
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#create-a-workflow-dispatch-event
+func (s *ActionsService) CreateWorkflowDispatchEventByID(ctx context.Context, owner, repo string, workflowID int64, event *CreateWorkflowDispatchEventRequest) (*Response, error) {
```

Comment:
We typically pass mandatory fields by value in this repo... so I'm thinking that these two exported functions should be left as they were and not use pointers.

However, I'm totally fine to let your `create...` helper function use a pointer since that stays internal to the package.

Reply:
I actually took a little hint from `CreateUserImpersonation` from admin_users.go that expects pointer to `ImpersonateUserOptions`. I will update the two funcs to expect value though.